approved.yaml

```diff
@@ -1004,6 +1004,10 @@ udlpasswall:
   - "UDLPasswall"
   - "Dynamic Lighting"
 
+metascripttoolbox:
```

Comment:
We will update this file when we do a release, please avoid updating this file in the future.

Reply:
Of course.

FYI, people update this because the merge has not always been consistently handled. It occurs on different days, or sometimes not at all... and specific to the YAML, addition to the 1-click has not always automatically followed a new script being merged into the repo... even if that is specifically asked for as a part of the branch notes. Apologies if editing the YAML is an annoyance, however it is the only way to (relatively) ensure that the person responsible for the git merge takes note that this is a new script.